Tasks/BashV3/bash.ts

```diff
@@ -94,14 +101,29 @@ async function run() {
             else {
                 targetFilePath = input_filePath;
             }
+
+            const [processedArgs, telemetry] = processBashEnvVariables(input_arguments)
+
+            const resultArgs = featureFlags.enableSecureArgs ? processedArgs : input_arguments
+
+            if (featureFlags.enableTelemetry) {
+                emitTelemetry('TaskHub', 'Bash', telemetry)
+            }
+
+            const argsEnvVar = {
+                envName: "INPUT_SCRIPT_ARGS",
+                value: resultArgs.trim()
+            };
+            process.env[argsEnvVar.envName] = argsEnvVar.value;
```

Comment:
>process.env [](http://example.com/codeflow?start=12&length=11)

if enableTelemetry is not enabled, can we avoid setting INPUT_SCRIPT_ARGS env variable?  (reduces risk if enableTelemetry is not enabled)

Reply:
Fixed, thanks!